qiskit/aqua/algorithms/minimum_eigen_solvers/qaoa/qaoa.py

```diff
@@ -158,3 +158,35 @@ def operator(self, operator: Union[OperatorBase, LegacyBaseOperator]) -> None:
                                     self._p,
                                     initial_state=self._initial_state,
                                     mixer_operator=self._mixer)
+
+    @property
+    def initial_state(self) -> Optional[Union[QuantumCircuit, InitialState]]:
```

Comment:
Should we move this to VQE instead of QAOA?

Reply:
There's no `initial_state` in VQE.

Reply:
oh, I see, then ignore my comment...